src/App/App.tsx

```diff
@@ -111,6 +111,9 @@ if (GA_CODE) {
 export interface AppState {
   username?: string;
 }
+interface AppProps {
+  logoutComponent?: (props: LogoutProps) => null;
```

Comment:
- Does this need to be optional?
- Can we add docstrings to this file please?

Reply:
@kahummer I think the idea was having a defualt value defined on APP for `logoutComponent`. On the other hand and this is my solo opinion, we don't have to supply the logout component as a prop to `APP`. I would argue its not like we are calling `APP` in more than a single place such that we would need to customize its props.

Reply:
@p-netm I think we can have it as a default value. It eased mocking the component when running test related to logout on App

Reply:
ok